pages/docs/tutorials/javascript/working-with-modules/working-with-modules.md

```diff
@@ -12,13 +12,13 @@ showAuthorInfo: false
 In this tutorial we'll see how to
 
 * [Configure modules when using IntelliJ IDEA](#configuring-modules-with-intellij-idea)
-* [Configure modules when using Maven or Gradle](#configuring-modules-when-using-maven-or-gradle)
+* [Configure modules when using Gradle](#configuring-modules-when-using-gradle)
 * [Use Kotlin in the browser with AMD](#using-amd)
 * [Use Kotlin from node.js with CommonJS](#using-commonjs)
 
 
 
-## Configuring Modules with IntelliJ IDEA
+## Configuring modules with IntelliJ IDEA
```

Comment:
Do we sure about content of this page?
It seems a bit outdated. Maybe we can at least adopt it for browser, when we use webpack.
Because the main core of article is the same, but samples may use webpack for build js library.